anchorecli/cli/__init__.py

```diff
@@ -8,6 +8,7 @@
 
 
 @click.group(context_settings=dict(help_option_names=['-h', '--help', 'help']))
+@click.option('--config', help='Set the location of the anchore-cli yaml configuration file')
```

Comment:
This needs to be either 'c' or change the input param to the function to be 'config' instead of 'c'. There isn't any auto-shortening. That is causing the build error in CI.